tests/test_haplotypecaller_skip_filter.yml

```diff
@@ -71,9 +75,9 @@
     # binary changes md5sums on reruns
     - path: results/variant_calling/haplotypecaller/test/test.haplotypecaller.vcf.gz.tbi
     # binary changes md5sums on reruns
-    - path: results/variant_calling/haplotypecaller/test/test.haplotypecaller.vcf.gz
-    # binary changes md5sums on reruns
-    - path: results/variant_calling/haplotypecaller/test/test.haplotypecaller.vcf.gz.tbi
-    # binary changes md5sums on reruns
```

Comment:
this should still exist

Reply:
Those lines were redundant duplicates. (Probably some copy-paste operation gone astray.)